python/ray/tests/test_channel.py

```diff
@@ -36,6 +36,10 @@ def test_put_local_get(ray_start_regular):
 def test_set_error_before_read(ray_start_regular):
     @ray.remote
     class Actor:
+        def create_channel(self, writer, readers):
```

Comment:
nit: We only call this function on the writer actor, so could omit the `writer` arg for create_channel() and just pass `this` to the Channel constructor

Reply:
Yes, there is a TODO for this, but I wanted to make this PR more narrow.

We should clean up this API but for now these are all experimental APIs, and Channel is a DeveloperAPI.